owrrm/src/main/java/com/facebook/openwifi/rrm/optimizers/clientsteering/ClientSteeringOptimizer.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.openwifi.rrm.optimizers.clientsteering;
+
+import java.util.Map;
+
+import com.facebook.openwifi.rrm.DeviceConfig;
+import com.facebook.openwifi.rrm.DeviceDataManager;
+import com.facebook.openwifi.rrm.modules.Modeler.DataModel;
+
+/** Client steering base class */
+public abstract class ClientSteeringOptimizer {
+
+	/** Represents the client steering related actions an AP can take */
+	public static final String STEER_UP_ACTION = "STEER_UP";
+	public static final String STEER_DOWN_ACTION = "STEER_DOWN";
```

Comment:
you may want to think about implementing in a way that will support more than 2 bands?